spec/openapi.yaml

```diff
@@ -37,6 +37,21 @@ paths:
           $ref: '#/components/responses/Created'
         '400':
           $ref: '#/components/responses/BadRequest'
+  /logs/:id:
+    get:
+      parameters:
+      - name: id
+        in: path
+        required: true
+        schema:
+          type: number
+      responses:
+        '200':
+          $ref: '#/components/responses/Created'
```

Comment:
The OpenAPI description of this response will be `Created` while this route only returns data, another response should fit  better.

Reply:
Fair enough, I propose to call it `OK` based on https://developer.mozilla.org/en-US/docs/Web/HTTP/Status

Reply:
(which would be consistent with 201 Created)